docs/users-guide/interactive-dashboards.md

```diff
@@ -0,0 +1,92 @@
+## Interactive dashboards
+
+You can customize what happens when people click on questions in your dashboard. You can set up a question to:
```

Comment:
Feels like this needs more of a lead-in. Maybe we should set the context by referencing what Metabase does by default on-click — i.e., we could link to the drill-through documentation.

E.g., "By default, when you create charts using Metabase's graphical query builder, your charts automatically come with drill-through capabilities, which let folks click on a chart to explore further. But if you have a more customized click path in mind, Metabase also allows you to customize what happens when a chart or table in a dashboard is clicked. You can set up a dashboard card to:"